CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+cmake_minimum_required(VERSION 3.0.0 FATAL_ERROR)
```

Comment:
What's the reason for adding a CMakeLists.txt file? Files that provide functions are usually included directly (see standard stuff that CMake provides). CMakeLists.txt files are supposed to add targets to the current project.

Reply:
Purpose of CMakeLists.txt is too include the script when using the FetchContent Declare/MakeAvailable.
Example : https://github.com/OlivierLDff/QaterialGallery/blob/master/platform/PostBuildPlatform.cmake